### ISSUE TITLE
[Feature Request] ZPL playlist Support

### ISSUE THREAD

**Issue by LAKSHAY184** [jagrosh/MusicBot#585]
**Is your feature request related to a problem? Please describe.**
I want to use a groove music playlist on a discord bot.
**Describe the solution you'd like**
Since, Groove stores the playlists in zpl files, it would be easy to just upload the zpl file in the folder and get it running.

**Describe the basic flow/steps of using this feature**
How this feature would be used:
1. this could be used to run groove music playlists
2. 
3. 

**Additional context**

**Have you confirmed that this idea is not already described here?**
https://github.com/jagrosh/MusicBot/projects/1
This feature is not here

**Comment by pbanj** [comment]
You can probably just change the file extension

**Comment by LAKSHAY184** [comment]
No, It can't be done because it is written in a zpl format which looks a bit similar to xml

**Comment by pbanj** [comment]
Ah, https://gist.github.com/jgarza9788/a17de65f41f14662491e this might be helpful for the time being.

**Comment by LAKSHAY184** [comment]
Thanks!

**Comment by LAKSHAY184** [comment]
> Ah, https://gist.github.com/jgarza9788/a17de65f41f14662491e this might be helpful for the time being.

hi! I was after some hard work able to change my playlist to m3u but it isn't working with the music bot
Can you guide me!
